src/Slider/Slider.js

```diff
@@ -191,7 +191,7 @@ class Slider extends PureComponent {
 
     if (found) {
       this.active = handleID
-      onSlideStart(values.map(d => d.val))
+      onSlideStart(values.map(d => d.val), {activeHandleID})
```

Comment:
I didn't try to run this locally, but it looks activeHandleID  would be undefined, right?  Shouldn't that be like...

```js
onSlideStart(values.map(d => d.val), { activeHandleID: handleID })      
```